src/bug_report.rs

```diff
@@ -213,6 +213,7 @@ fn get_config_path(shell: &str) -> Option<PathBuf> {
             "elvish" => Some(".elvish/rc.elv"),
             "tcsh" => Some(".tcshrc"),
             "xonsh" => Some(".xonshrc"),
+            "cmd" => Some("starship.lua"),
```

Comment:
I think this is the usual location:
```suggestion
            "cmd" => Some("AppData/Local/clink/starship.lua"),
```
But maybe this should be set to `.inputrc` or `AppData/Local/clink/clink_settings`?

Reply:
> I think this is the usual location:

I left out the `AppData/Local/clink/` part because this location is configurable in multiple ways (through `clink.path` setting, or through `clink installscripts` command)

> But maybe this should be set to `.inputrc` or `AppData/Local/clink/clink_settings`?

I don't this is relevant for Starship. FWIW, .inputrc is also there for bash, but we don't use it. Similarly clink_settings file mostly stores stuff other than prompt (except transience, which we removed in this PR). `AppData/Local/clink/starship.lua` is a better fit.

Reply:
> I left out the `AppData/Local/clink/` part because this location is configurable in multiple ways (through `clink.path` setting, or through `clink installscripts` command)

I know, but I suggested this because the other option was even more likely to be wrong.

> I don't think this is relevant for Starship. FWIW, .inputrc is also there for bash, but we don't use it. Similarly clink_settings file mostly stores stuff other than prompt (except transience, which we removed in this PR). `AppData/Local/clink/starship.lua` is a better fit.

Thanks for this info. I wasn't very sure what each of these files was responsible for.